nix/dev-vm/default.nix

```diff
@@ -0,0 +1,28 @@
+# NOTE: modulesPath and imports are taken from nixpkgs#59219
+{ modulesPath, pkgs, lib, ... }: {
+  imports = [ (modulesPath + "/virtualisation/qemu-vm.nix") ];
+  services.openssh = {
+    enable = true;
+    permitRootLogin = "yes";
+  };
+
+  services.docspell-restserver = {
```

Comment:
@eikek currently  when the VM starts, one needs to register every time. Is there any way to declaratively create users for testing? Two ways I could think of:

* Create users through API calls.
* Postgres way:
  1. Move development VM to postgres
  2. Directly manipulate the DB creating users

But the second approach seems quite brittle.

Reply:
I think doing it via api would be easier. Would be nice if dsc could do that :). Another way could be to create a dump file and restore it via pg_restore.

Reply:
I reused the generated qcow2 file between runs - worked for now :)